src/views/LoggingRoom/LogMessageRoom.vue

```diff
@@ -30,9 +31,7 @@
             ></BaseInput>
```

Comment:
BaseInput is a deprecated component, can you please change this to use `b-input` instead?